### PR TITLE
WW-361 buttons styling

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/e7a45bb87f6b771de6671da573af8a978ea4d663/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/e7a45bb87f6b771de6671da573af8a978ea4d663/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L433](https://github.com/Wikia/app/blob/e7a45bb87f6b771de6671da573af8a978ea4d663/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L433): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. MAJOR [WikiBuilder.js#L584](https://github.com/Wikia/app/blob/e7a45bb87f6b771de6671da573af8a978ea4d663/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L584): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
1. MAJOR [WikiBuilder.js#L586](https://github.com/Wikia/app/blob/e7a45bb87f6b771de6671da573af8a978ea4d663/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L586): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Comment by sebastianmarzjan** [comment]
Are you sure you want this PR to dev and not to a feature branch so the changes are more precise?

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 1]
```diff
@@ -54,30 +56,27 @@ $color-borders: #CECECE;
...
-				float: left;
-				margin-right: 10px;
-			}
+		input[type=button] {
+			@extend .wds-button;
+			@extend .wds-is-text;
+			background: white;
+			float: right;
+			height: auto;
+			margin: 6px 0 6px 6px;
```
Comment (sebastianmarzjan) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: Can we use content-spacing vars for that?

File: extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss [hunk 2]
```diff
@@ -54,30 +56,27 @@ $color-borders: #CECECE;
...
-			}
-
-			.error-msg {
-				float: left;
-				margin-right: 10px;
-			}
+		input[type=button] {
+			@extend .wds-button;
+			@extend .wds-is-text;
+			background: white;
```
Comment (sebastianmarzjan) [inline extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss]: Can we use a color var for this?

**Comment by sebastianmarzjan** [comment]
Two minor things related to using SASS vars, rest seems fine to me

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [WikiBuilder.js#L9](https://github.com/Wikia/app/blob/db5c4e5e6a1bec8a2ada34badb48409e7a8deab0/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L9): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
2. MAJOR [WikiBuilder.js#L10](https://github.com/Wikia/app/blob/db5c4e5e6a1bec8a2ada34badb48409e7a8deab0/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L10): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
3. MAJOR [WikiBuilder.js#L433](https://github.com/Wikia/app/blob/db5c4e5e6a1bec8a2ada34badb48409e7a8deab0/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L433): Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
4. MAJOR [WikiBuilder.js#L584](https://github.com/Wikia/app/blob/db5c4e5e6a1bec8a2ada34badb48409e7a8deab0/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L584): Remove this useless assignment to local variable "cityId" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)
5. MAJOR [WikiBuilder.js#L586](https://github.com/Wikia/app/blob/db5c4e5e6a1bec8a2ada34badb48409e7a8deab0/extensions/wikia/CreateNewWiki/js/WikiBuilder.js#L586): Remove this useless assignment to local variable "createStatusMessage" [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)

**Review by lukaszkonieczny — APPROVED** [review]

**Pull request merged**